go/vt/vtgate/executor_framework_test.go

```diff
@@ -555,3 +556,9 @@ func testQueryLog(t *testing.T, logChan chan interface{}, method, stmtType, sql
 
 	return logStats
 }
+
+func newTestResolver(hc discovery.HealthCheck, serv srvtopo.Server, cell string) *Resolver {
```

Comment:
Where is this being used?

Reply:
In this same file. The executor still uses couple of functions from the resolver, which will be fixed by @systay and @harshit-gangal soon.